admin/includes/modules/search_box.php

```diff
@@ -37,16 +37,18 @@
     <?php
     if (file_exists($searchBoxJs)) {
         ?>
-        <div class="form-group" id="searchTermRepopulate">
-            <div class="col-sm-3 control-label"><?= TEXT_INFO_SEARCH_FILTER_REPOPULATE; ?></div>
-            <div class="col-sm-9">
-                <?= zen_draw_checkbox_field('repopulateSearch', '', (!empty($_GET['repopulateSearch']) && $_GET['repopulateSearch'] === 'on'), '', ' id="repopulateSearch"'); ?>
+        <div class="row">
+            <div class="form-group col-md-4" id="searchTermRepopulate">
+                <div class="col-sm-9 control-label"><?= TEXT_INFO_SEARCH_FILTER_REPOPULATE; ?></div>
+                <div class="col-sm-1">
+                    <?= zen_draw_checkbox_field('repopulateSearch', '', (!empty($_GET['repopulateSearch']) && $_GET['repopulateSearch'] === 'on'), '', ' id="repopulateSearch"'); ?>
+                </div>
             </div>
```

Comment:
In order for the checkbox's label to also be clickable to toggle the checkbox, the HTML needs to be of the following form:
```
        <div class="checkbox">
            <label>
                <?php echo zen_draw_checkbox_field('repopulateSearch', '', (!empty($_GET['repopulateSearch']) && $_GET['repopulateSearch'] === 'on'), '', ' id="repopulateSearch"'); ?>
                <?php echo ' ' . TEXT_INFO_SEARCH_FILTER_REPOPULATE; ?>
            </label>
        </div>
```
Granted, I've provided neither a col-xx-yy class nor the 'id' attribute, but that's the basic Bootstrap structure that's required for the label to act a the checkbox toggle-extension.

Ditto for the second checkbox, below.